src/js/smooch.jsx

```diff
@@ -161,6 +164,14 @@ export class Smooch {
                 }
             });
         }).then((loginResponse) => {
+            observable.on('socket:connected', () => {
```

Comment:
Also, is it useful to publish that on the public event hub.. And why use events instead of calling the necessary functions directly... I don't want to fall back in the events trap Backbone and Marionette had us do.